src/routes/match.ts

```diff
@@ -0,0 +1,57 @@
+import { Router } from 'express';
+import { Queue } from '../match/queue';
+import User from '../models/user';
+const router = Router();
+
+const unrankedQueue = new Queue();
+// join ranked queue API
+router.post('/join', async (req, res) => {
+  const { username }: { username: string } = req.body;
+  if (!username) {
+    return res.status(400).send({ message: 'cad request' });
+  }
+  const usernameLower = username.toLowerCase();
+  const isPlayerExist = await User.find({
```

Comment:
Can do `findOne()` here and simply check `if(user)`

`const user = await User.findOne({})`.